### DISCUSSION TITLE
Change the Shape of View Field

### DISCUSSION THREAD

**Discussion by UCmay** [sn-lab/MouseGoggles#17] [Q&A]
Sorry to bother you again.
I have trouble with Godot. I want to change the shape of the view field. My new screen is rectangular. 
![rec](https://github.com/sn-lab/mouseVRheadset/assets/140501075/4b46634b-2061-4f0a-b3d8-6cb395587820)
Thank you!
Oh, I am sorry, I fix it. Sorry again, that's really a dumb question.

**Comment by misaacson01** [comment]
Rectangular screens, which ones are you using?
